include/config.h

```diff
@@ -19,6 +19,8 @@ enum mako_button_binding {
 };
 
 struct mako_config {
+	// Configurable per notification
```

Comment:
Maybe we should add a `struct mako_notification_config` and add a `struct mako_notification_config default` field to `mako_config`

Reply:
I've been debating that also. It really depends on how many things we expect to accumulate over time that don't make sense to override per-notification. (Hence the item about max_visible in the "things I'm not sure about" list.)